readme.md

```diff
@@ -8,7 +8,11 @@ Please find more info about each part in the relevant Readme file ([frontend](fr
 
 When implementing a new feature or fixing a bug, please create a new pull request against `main` from a feature/bug branch and add `@vanessa-cooper` as reviewer.
 
-## How to run locally?
+## How to run in dev mode?
+
+### Using Codespace
+1.  run `docker-compose up`
+### On your local machine
```

Comment:
what's that?

Reply:
Well just noticed it along the way. Users that are using codespace don't need to  download docker